extensions/amp-brightcove/1.0/component.js

```diff
@@ -37,6 +37,8 @@ export function BrightcoveWithRef(props, ref) {
     videoId,
     onPlayingState,
     urlParams,
+    onLoad,
+    // onError, // todo: determine when to invoke
```

Comment:
oh whoops, i'd meant to submit a comment about this asking how you think we should handle this.

yeah i had trouble even simulating an error because most of these components handle "invalid" urls gracefully, so I'll just remove all the references to them.

Reply:
removed